scripts/setup/setup-certbot

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -e
+
+usage() {
+  echo "Usage: $0 <domain name> <email>" >&2
+  exit 1
+}
+
+if [ "$EUID" -ne 0 ]; then
+    echo "Error: This script must be run as root" >&2
+    exit 1
+fi
+
+args="$(getopt -o '' --long help,hostname:,email: -n "$0" -- "$@")"
+eval "set -- $args"
+while true; do
+    case "$1" in
+        --hostname)
+            DOMAIN="$2"
+            shift
+            shift
+            ;;
+        --email)
+            EMAIL="$2"
+            shift
+            shift
+            ;;
+        --help)
+            show_help=1
+            shift
+            ;;
+        --)
+            break
+            ;;
+    esac
+done
+
+if [ -n "$show_help" ]; then
+    usage
+fi
+
+# For reference https://certbot.eff.org/all-instructions/#debian-other-nginx
+wget https://dl.eff.org/certbot-auto
+chmod a+x certbot-auto
+
+./certbot --nginx certonly -d "$DOMAIN" -m "$EMAIL" --agree-tos --non-interactive
```

Comment:
Having the `--nginx` flag here will have no effect since the conf file will be overwritten anyway. But I leave here for the possibility that overwriting the nginx conf is to be replaced with `meld`.